pascal.html.markdown

```diff
@@ -66,7 +66,7 @@ program Learn_More;
 
 const
     PI = 3.141592654;
-    GNU = 'GNU's Not Unix';
+    GNU = 'GNU''s Not Unix';
```

Comment:
Is two single quotes a common way to escape a single quote in Pascal? Just asking because not all languages have the same syntax for that.

Reply:
It is. I was expecting `\` but that is not the case.